saucedemo_framework/tests/test_cart_page.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+@pytest.mark.regression
+def test_continue_shopping(open_cart_page):
+    cart_page = open_cart_page
+    store_page = cart_page.continue_shopping()
+    assert store_page.is_secondary_header_located() is True, 'Wasn`t able to continue shopping'
+
+
+@pytest.mark.regression
+def test_remove_item_from_cart(open_cart_page_with_item):
+    cart_page = open_cart_page_with_item
+    cart_page.press_remove_button()
+    assert cart_page.is_remove_button_displayed() is False, 'Item was not removed from the cart'
```

Comment:
this verification is not good. You should check if your item still in cart

Reply:
Done